notepadqq-1.4.8-xdg.patch

```diff
@@ -0,0 +1,281 @@
+ 1 file changed, 1 deletion(-)
+
+diff --git a/support_files/notepadqq.appdata.xml b/support_files/notepadqq.appdata.xml
+index f8dae7a4e..10e8742f4 100644
+--- a/support_files/notepadqq.appdata.xml
++++ b/support_files/notepadqq.appdata.xml
+@@ -4,7 +4,6 @@
+   <metadata_license>MIT</metadata_license>
+   <project_license>GPL-3.0</project_license>
+   <name>Notepadqq</name>
+-  <icon width="96" height="96">https://user-images.githubusercontent.com/4319621/36906314-e3f99680-1e35-11e8-90fd-f959c9641f36.png</icon>
+   <summary>An advanced text editor</summary>
+   <description>
+     <p>
+
+From 6b80d126341243f84eb02a1863320d1bc4b9bd53 Mon Sep 17 00:00:00 2001
+From: scx <scx.mail@gmail.com>
+Date: Sat, 1 Dec 2018 20:18:15 +0100
+Subject: [PATCH 2/7] chore: Enhance AppData file (#808)
+
+---
+ README.md                           | 2 +-
+ support_files/notepadqq.appdata.xml | 8 +++++++-
+ 2 files changed, 8 insertions(+), 2 deletions(-)
+
+diff --git a/README.md b/README.md
+index 0824c55ab..c756a0f85 100644
+--- a/README.md
++++ b/README.md
+@@ -8,7 +8,7 @@
+ 
+ #### What is it?
+ 
+-Notepadqq is a text editor designed from developers, for developers. 
++Notepadqq is a text editor designed by developers, for developers. 
+ 
+ ![screenshot_20180302_163505](https://notepadqq.com/s/images/snapshot1.png)
+ 
+diff --git a/support_files/notepadqq.appdata.xml b/support_files/notepadqq.appdata.xml
+index 10e8742f4..5f0d67552 100644
+--- a/support_files/notepadqq.appdata.xml
++++ b/support_files/notepadqq.appdata.xml
+@@ -1,4 +1,4 @@
+- <?xml version="1.0" encoding="UTF-8"?>
++<?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+   <id>notepadqq.desktop</id>
+   <metadata_license>MIT</metadata_license>
+@@ -28,4 +28,10 @@
+   <url type="homepage">https://github.com/notepadqq/notepadqq</url>
+   <url type="bugtracker">https://github.com/notepadqq/notepadqq/issues</url>
+   <url type="help">https://github.com/notepadqq/notepadqq/wiki</url>
++  <provides>
++    <binary>notepadqq</binary>
++  </provides>
++  <releases>
++    <release version="1.4.8" date="2018-05-10"/>
++  </releases>
+ </component>
+
+From 3c16a4134855639ff23b3275a0ceac3dcc30a2da Mon Sep 17 00:00:00 2001
+From: scx <scx.mail@gmail.com>
+Date: Mon, 10 Dec 2018 15:41:49 +0100
+Subject: [PATCH 3/7] chore: add OARS metadata (#811)
+
+---
+ support_files/notepadqq.appdata.xml | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/support_files/notepadqq.appdata.xml b/support_files/notepadqq.appdata.xml
+index 5f0d67552..e00dcb8e5 100644
+--- a/support_files/notepadqq.appdata.xml
++++ b/support_files/notepadqq.appdata.xml
+@@ -28,6 +28,9 @@
+   <url type="homepage">https://github.com/notepadqq/notepadqq</url>
+   <url type="bugtracker">https://github.com/notepadqq/notepadqq/issues</url>
+   <url type="help">https://github.com/notepadqq/notepadqq/wiki</url>
++  <content_rating type="oars-1.0">
++    <content_attribute id="social-info">moderate</content_attribute>
```

Comment:
Curious what that would be?

Reply:
@TingPing 
> Curious what that would be?

Optional (disabled by default) collecting anonymous statistics.
https://github.com/notepadqq/notepadqq/blob/v1.4.8/src/ui/frmpreferences.ui#L115-L119
> Collect and transmit anonymous statistics to improve Notepadqq

You already know that. We've talked about this.